controller/src/bin/xcvradm.rs

```diff
@@ -291,6 +351,22 @@ enum Cmd {
         /// a "1" is printed. Otherwise a "0" is printed.
         #[arg(long)]
         all: bool,
+
+        /// Print the output in a parseable format.
+        #[arg(long, short, conflicts_with_all = ["all", "with", "without"])]
+        parseable: bool,
```

Comment:
The fields `parseable`, `output`, `output_separator`, and `omit_header` are repeated a bunch. I don't think clap plays nicely with generics, but is there a way to factor this out and then `#[command(flatten)]` it into place?